node/options.go

```diff
@@ -500,13 +501,13 @@ func WithRequestHandler(requestHandler process.RequestHandler) Option {
 	}
 }
 
-// WithAntifloodHandler sets up an antiflood handler for the Node
-func WithAntifloodHandler(antifloodHandler P2PAntifloodHandler) Option {
+// WithInputAntifloodHandler sets up an antiflood handler for the Node on the input side
+func WithInputAntifloodHandler(antifloodHandler P2PAntifloodHandler) Option {
```

Comment:
WithAntifloodHandler was a good enough name

Reply:
We now have both outputAntifloodHandler and inputAntifloodHandler. This was changed as to signal that the node will use (for now) the input antiflood handler